src/storage.ts

```diff
@@ -10,14 +10,14 @@
 
 interface PersistedData {
     hasPermission: boolean;
-    lastUsedCameraId: string | null;
+    lastUsedCameraId: string;
```

Comment:
This one was intentionally `string | null` to indicate that the `lastUserCameraId` could be unset.

Is there a guideline / rationale to prefer empty string over `null`?

Reply:
You are right. It's better to define a null value.